docs/en_US/Release.md

```diff
@@ -1,5 +1,46 @@
 # ChangeLog
 
+## Release 1.5 - 4/13/2020
+
+### New Features and Documentation
+
+#### Hyper-Parameter Optimizing
+
+* New tuner: [Population Based Training (PBT)](https://github.com/microsoft/nni/blob/master/docs/en_US/Tuner/PBTTuner.md)
+* Trials can now report infinity, NaN, and string metrics
```

Comment:
what does `string metrics` mean?

Reply:
Metrics of string type. Maybe "trials can now report infinity and NaN as result, as well as metrics in string format"?